test/DataQI.Commons.Test/Util/AssertTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Xunit;
+
+using AssertUtil = DataQI.Commons.Util.Assert;
+
+namespace DataQI.Commons.Test.Util
+{
+    public class AssertTest
+    {
+        [Fact]
+        public void TestAssertIsType()
+        {
+            var expectedMessage = "Invalid type";
+
+            var objOne = new TypeOne();
+            var objTwo = new TypeOne();
+
+            AssertUtil.IsType(objOne.GetType(), objTwo, expectedMessage);
+        }
+
+        [Fact]
+        public void TestAssertIsTypeThrowsException()
```

Comment:
Don't fluent assertions cover this? Also if not a good idea for a small lib xD

Reply:
Yes, It does!
However, I prefer to avoid dependence on third parties at this time and for as long as possible.